### PR TITLE
Syntax fixes and Grammarly recommendations

### PR DESCRIPTION
**Pull request by joebordes**
While translating the ES strings I ran into some errors and recommendations I thought I could share.

HTH

### CODE REVIEW

File: app/bundles/AssetBundle/Translations/en_US/messages.ini [hunk 1]
```diff
@@ -19,7 +19,7 @@ mautic.asset.asset.form.storageLocation.local="Local"
 mautic.asset.asset.form.storageLocation.remote="Remote"
 mautic.asset.asset.form.disallow.crawlers="Block search engines from indexing this file"
 mautic.asset.asset.form.disallow.crawlers.descr="If you don't want to index files like PDF, DOCX etc, this option will disallow search bots by using the X-Robots-Tag HTTP header."
-mautic.asset.asset.help.alias="Letters and numbers (hyphens allowed) used for URL generation of this asset. A unique alias based on the title will be autogenerated if left empty."
+mautic.asset.asset.help.alias="Letters and numbers (hyphens allowed) are used for URL generation of this asset. A unique alias based on the title will be autogenerated if left empty."
```
Comment (andersonjeccel) [inline app/bundles/AssetBundle/Translations/en_US/messages.ini]: This one won't make sense in the context where it's used

File: app/bundles/CategoryBundle/Translations/en_US/messages.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 mautic.category.createnew="Create new category..."
-mautic.category.form.alias.help="Letters and numbers (hyphens allowed) used for URL generation of this category. A unique alias based on the title will be autogenerated if left empty."
+mautic.category.form.alias.help="Letters and numbers (hyphens allowed) are used for URL generation of this category. A unique alias based on the title will be autogenerated if left empty."
```
Comment (andersonjeccel) [inline app/bundles/CategoryBundle/Translations/en_US/messages.ini]: This one won't make sense in the context where it's used

File: app/bundles/PageBundle/Translations/en_US/messages.ini [hunk 3]
```diff
@@ -79,7 +79,7 @@ mautic.page.graph.pie.new.vs.returning.returning="Returning"
 mautic.page.graph.pie.time.on.site="Time on site"
 mautic.page.header.edit="Edit Page - %name%"
 mautic.page.header.new="New Page"
-mautic.page.help.alias="Letters and numbers (hyphens allowed) used for URL generation of this landing page. A unique alias based on the title will be autogenerated if left empty."
+mautic.page.help.alias="Letters and numbers (hyphens allowed) are used for URL generation of this landing page. A unique alias based on the title will be autogenerated if left empty."
```
Comment (andersonjeccel) [inline app/bundles/PageBundle/Translations/en_US/messages.ini]: This one won't make sense in the context where it's used

**Review by andersonjeccel — CHANGES_REQUESTED** [review]
Hi Joe,
First things first: Thank you for creating this PR and addressing these changes, they're useful and help us to polish Mautic :)

I left 3 comments where the changes would become a bit strange as the messages are describing a feature instead of talking about its capabilities.
Would you be up for reverting these?

File: app/bundles/AssetBundle/Translations/en_US/messages.ini (hunk 1 above)
Comment (joebordes) [reply app/bundles/AssetBundle/Translations/en_US/messages.ini]: reverted

File: app/bundles/CategoryBundle/Translations/en_US/messages.ini (hunk 2 above)
Comment (joebordes) [reply app/bundles/CategoryBundle/Translations/en_US/messages.ini]: reverted

File: app/bundles/PageBundle/Translations/en_US/messages.ini (hunk 3 above)
Comment (joebordes) [reply app/bundles/PageBundle/Translations/en_US/messages.ini]: reverted

**Comment by joebordes** [comment]
happy to help :-)

I made the changes you requested. let me know if you need any additional changes.

**Review by andersonjeccel — APPROVED** [review]
Perfect now, thanks for your contribution!

**Pull request merged**